symphony/lib/boot/func.utilities.php

```diff
@@ -8,32 +8,32 @@
  * Redirects the browser to a specified location. Safer than using a
  * direct header() call
  *
- *  @param string $url
+ *  @param string $location
+ *  @param string $status (optional)
```

Comment:
@jensscherbl: Do I remember correctly? It looks to me as if GitHub lost some comments here (probably because of the recent attack). Am I right?